examples/RM3NewPTOSim/RM3Files/powerPlotCode.m

```diff
@@ -0,0 +1,26 @@
+TimeV = PTOSimBlock5_out.time;
+shaftSpeed = PTOSimBlock5_out.signals.values(:,1);
+TorqueHM = PTOSimBlock5_out.signals.values(:,2);
+Current = PTOSimBlock6_out.i.Data;
+Voltage = PTOSimBlock6_out.Voltage.Data;
+
```

Comment:
@jleonqu Just for clarification, but the numeric values after PTOSimBlock_#_ will not be automatically updated and in this example these PTOSimBlocks are the corresponding blocks for the power conversion process. If so you may want to clarify as comments that if changes are made to this `powerPlotCode.m` the user will need to update these manually.